app/routes/events/components/EventAdministrate/Allergies.tsx

```diff
@@ -113,14 +120,16 @@ const Allergies = ({
       })
     : initialColumns;
 
+  console.log(event);
+
   const numOfAllergies = () => {
     return registered.filter(
       (registration) => registration.user.allergies?.length !== 0
     ).length;
   };
   return (
     <>
-      {currentUser.id === event.createdBy ? (
+      {canSeeAllAllergies(currentUser, event) ? (
```

Comment:
Again, use the actionGrant, since this is already determined on the backend

Reply:
Just currently struggling with getting the actionGrant to be passed with the administrateEvent